src/lib/journey/journey.svelte

```diff
@@ -20,14 +22,21 @@
 </script>
 
 {#if !$journeyStore?.completed}
-  <svelte:component
-    this={mapStepToStage($journeyStore.step)}
-    failureMessage={$journeyStore.error?.message}
-    bind:formEl
-    loading={$journeyStore.loading}
-    {submitForm}
-    step={$journeyStore.step}
-  />
+  {#if !$journeyStore.step}
```

Comment:
This was moved here from the stage components. It reduces some of the duplication and simplifies the stage components a bit.